core/auth/fake/module_test.go

```diff
@@ -0,0 +1,34 @@
+package fake
+
+import (
+	"testing"
+
+	"flamingo.me/flamingo/v3/framework/config"
+)
+
+func TestModule(t *testing.T) {
+	t.Parallel()
+	c := config.Map{
+		"flamingo.debug.mode": true,
+		"fake": config.Map{
```

Comment:
This is outdated / not relevant anymore

Reply:
`fake` path is not correct

Reply:
ok, removed and maybe fixed - we'll have a look on this test later again. It seems to test something else.